docs/cep/sink/index.md

```diff
@@ -37,14 +37,7 @@ The attribute names in the double curly braces are replaced with the values from
 
 The `type` parameter of the `sink.type` annotation defines the sink type that publishes the events. The other parameters of the `sink.type` annotation depends upon the selected sink type, and here some of its parameters can be optional and/or dynamic.
```

Comment:
```suggestion
The `type` parameter of the `sink.type` annotation defines the sink type that publishes the events. The other parameters of the `sink.type` annotation depends upon the selected sink type, and here some of its parameters can be optional and dynamic.
```

@diana-macrometa When replacing "and/or," any preference on going with "and" vs "or"?  I typically go with "and" since having both options available typically means you can pick one or the other, as well.

Reply:
Depends on the usage and the meaning of the sentence. Usually "and" is the best choice, but there are edge cases where it isn't.